extensions/amp-base-carousel/1.0/base-carousel.jss.js

```diff
@@ -17,7 +17,7 @@
 import {createUseStyles} from 'react-jss';
 
 const scrollContainer = {
-  position: 'absolute',
+  position: 'relative',
```

Comment:
If it's relative now, should we remove `top/left/right/bottom: 0`?

Reply:
Good catch, done.